sources/packages/backend/apps/api/src/auth/decorators/institution/institution-student-data-access.decorator.ts

```diff
@@ -10,5 +10,11 @@ export const INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY =
  * Provide context that it's consumer must be
  * validated to have access to data of given student.
  */
-export const HasStudentDataAccess = (studentIdParamName: string) =>
-  SetMetadata(INSTITUTION_HAS_STUDENT_DATA_ACCESS_KEY, studentIdParamName);
+export const HasStudentDataAccess = (
```

Comment:
Comments can be expanded as per the updated code.

Reply:
mm, I didn't change the comment, because it says `validated to have access to data of given student.`. and I was assuming that the application is also part of student data. But I can add details to it